src/compiler/checker.ts

```diff
@@ -6532,9 +6530,14 @@ module ts {
         }
 
         function getTypeAtPosition(signature: Signature, pos: number): Type {

```

Comment:
I had it that way, but this actually seemed better (and shorter). Anyway, I can change it back.